### ISSUE TITLE
Remove 'assertion_type' from _generate_refresh_request_body

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#238]
```
Remove 'assertion_type' from _generate_refresh_request_body [1].

Since it doesn't implement it's own to_json method, the value passed to the 
constructor will become irrelevant (as all the subclasses ignore it and use a 
constant).

Unfortunately, the signature of the parent class constructor can't be changed.

[1]: 
https://code.google.com/p/google-api-python-client/source/browse/oauth2client/cl
ient.py?r=9d5f47cc7ec138ca5eec114071f61e9733a14514#851

[2] 
https://code.google.com/p/google-api-python-client/source/browse/oauth2client/cl
ient.py?r=9d5f47cc7ec138ca5eec114071f61e9733a14514#939


https://code.google.com/p/google-api-python-client/source/detail?r=161133c247190
f8d81ee4f34ea0ff06d0bbb129b
```

Original issue reported on code.google.com by `dhermes@google.com` on 8 Feb 2013 at 9:43

**Comment by GoogleCodeExporter** [comment]
```
Issue 239 has been merged into this issue.
```

Original comment by `jcgregorio@google.com` on 11 Feb 2013 at 3:37

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 0b6b5de819ee.
```

Original comment by `dhermes@google.com` on 11 Feb 2013 at 4:46
- Changed state: **Fixed**
